src/SFA.DAS.ApprenticeCommitments.Api/Controllers/ApprenticeshipController.cs

```diff
@@ -39,6 +40,12 @@ public async Task<IActionResult> UpdateApprenticeship(UpdateApprenticeshipComman
             return Accepted();
         }
 
+        [HttpGet("/apprentices/{apprenticeId}")]
+        public async Task<IActionResult> GetApprentice(Guid apprenticeId)
+        {
+            return await new GetRequest<Apprentice>($"/apprentices/{apprenticeId}").Get(_client);
```

Comment:
This is an unusual pattern, I would have expected _client to be the format to be implemented in the same way as the shared code (Dan put together)